src/formats.c

```diff
@@ -1922,6 +1926,28 @@ static void test_fmt_split_unifies_case_4(struct fmt_main *format, char *ciphert
 		*is_split_unifies_case = 0;
 	return;
 }
+
+static char *test_fmt_tunable_costs( struct fmt_main *format)
+{
+	int i, j;
+	char *name;
+
+	for (i = 0; i < FMT_TUNABLE_COSTS; i++) {
+		if ((name = format->params.tunable_cost_name[ i])) {
+			if (!format->methods.tunable_cost_value[ i])
+				return "more tunable cost names than methods";
+			for (j = 0; name[ j] != '\0'; j++)
+				if (name[ j] == ',')
+					return "tunable cost name contains commas";
+			if (j == 0)
+				return "empty tunable cost name";
+
+		}
+		else if (format->methods.tunable_cost_value[ i])
```

Comment:
What's with the space before `i` and `j` within brackets everywhere? They seem to be intended.

Reply:
> What's with the space before `i` and `j` within brackets everywhere? They seem to be intended.

Fixed now